src/lang-promql/complete/promql.terms.ts

```diff
@@ -320,7 +320,16 @@ export const aggregateOpTerms = [
   },
 ];
 
-export const aggregateOpModifierTerms = [{ label: 'by' }, { label: 'without' }];
+export const aggregateOpModifierTerms = [
+  {
+    label: 'by',
+    info: 'Drop labels that are not listed in the "by" clause, even if their label values are identical between all elements of the vector',
```

Comment:
We could make this one simpler too:

```suggestion
    info: 'Keep the listed labels, remove all others.',
```